etc/microfw-on-failure.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=MicroFW failure state
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/systemctl stop keepalived
```

Comment:
@weratyr this effectively creates a dependency from MicroFW to keepalived, which I'd like to avoid.

I think you can achieve the same thing using a track script in keepalived. Add a stanza such as this to `keepalived.conf`:

```
vrrp_script check_microfw {
    script "systemctl -q status microfw"
    interval 2
}
```

And then, in your VRRP instance:

```
vrrp_instance VI_1 {
    interface eth1
    state MASTER
    priority 200
    # other stuff

    track_script {
        check_microfw
    }
}
```

This should achieve the same thing, wdyt?

Reply:
Yes, this sounds good to me. Honestly, I didn't know that keepalived is able of track_scripts. 
So the `microfw-on-failure.service` was a quick solution for my use case. 😅 


However, it seems I made a lot of stacked changes here, 
I will split up this PR and create for each topic a separate one.